ocr.py

```diff
@@ -294,7 +318,7 @@ def run_ocr(self, image):
         """
         start_time = time.time()
         output_text, highlighted_image, annotations, reg_result = self.ocr_and_highlight(image)
-        annotations = self.postprocess_bbox(annotations)
+        # annotations = self.postprocess_bbox(annotations)
```

Comment:
why is this commented and not removed? is this still used?

Reply:
This function is for combining overlapping bboxes.
It's only commented out because I don't quite like the results of it for now. but I still need to do postprocessing on the bbox results.